reactive/INTRODUCTION_EN.md

```diff
@@ -0,0 +1,153 @@
+# Introduction to Reactive Programming
+
+> Written by [A~Z](https://github.com/AZ-0)
+
+Do you know of reactive programming? No? You should!
+
+Let's begin with wikipedia's obscure definition:
+
+## Definition
+
+The original article has the [french](https://fr.wikipedia.org/wiki/Programmation_r%C3%A9active) definition as well, but seeing as it is quite useless I just removed it.
+You shouldn't need french when programming anyway, no?
+
+We only have the [english article](https://en.wikipedia.org/wiki/Reactive_programming) left then, so here it is:
+> In computing, reactive programming is a declarative programming paradigm concerned with data streams and the propagation of change.
+> With this paradigm it is possible to express static (e.g., arrays) or dynamic (e.g., event emitters) data streams with ease, and also communicate that an inferred dependency
+> within the associated execution model exists, which facilitates the automatic propagation of the changed data flow.
+
+Wow, that's a lot of cool and complicated words! Don't worry, at this point not feeling overwhelmed would be concerning.
+(Or it just means you actually don't need this introduction at all so stop reading it and do something useful with your life)
+
+## An explanation to rekindle lost hopes
+
+Looks like you have a bad karma, because I am gonna be mean~. So mean that I'll use complicated words right away: `Publisher`! (Are you fearing me now?)
```

Comment:
```suggestion
Looks like you have bad karma because I am gonna be mean~. So mean that I'll use complicated words right away: `Publisher`! (Do you fear me now?)
```